src/main/java/org/prebid/server/bidder/yieldlab/YieldlabBidder.java

```diff
@@ -408,4 +476,19 @@ private String makeNurl(BidRequest bidRequest, ExtImpYieldlab extImpYieldlab, Yi
                 yieldlabResponse.getAdSize(),
                 uriBuilder.toString().replace("?", ""));
     }
+
+    private ObjectNode resolveExtParameter(YieldlabDigitalServicesActResponse dsa) {
+        if (dsa == null) {
+            return null;
+        }
+        final ObjectNode ext = mapper.mapper().createObjectNode();
+        final JsonNode dsaNode;
+        try {
+            dsaNode = mapper.mapper().convertValue(dsa, JsonNode.class);
+        } catch (IllegalArgumentException e) {
+            throw new PreBidException("Failed to serialize DSA object", e);
```

Comment:
looks like the PBS Go version has a more detailed error message `fmt.Errorf("failed to make JSON for seatbid.bid.ext for adslotID %v. This is most likely a programming issue", bid.ID)`

Perhaps it's worth keeping it here as well

Also, according to the PBS Go version the error shouldn't disrupt other bids processing, so the error has to be reported but the ext has to be null in this case. Please double-check